lib/hsa/hsaPulsarUpdatePhase.cpp

```diff
@@ -50,14 +53,13 @@ hsaPulsarUpdatePhase::hsaPulsarUpdatePhase(Config& config, const string& unique_
     vector<float> dg = {0.0, 0.0}; // re,im
     default_gains = config.get_default<std::vector<float>>(unique_name, "frb_missing_gains", dg);
 
-    _source_ra = config.get<std::vector<float>>(unique_name, "source_ra");
-    _source_dec = config.get<std::vector<float>>(unique_name, "source_dec");
-    _source_scl = config.get<std::vector<int>>(unique_name, "psr_scaling");
-
-    for (int i = 0; i < _num_beams; i++) {
-        psr_coord_latest_update.ra[i] = _source_ra[i];
-        psr_coord_latest_update.dec[i] = _source_dec[i];
-        psr_coord_latest_update.scaling[i] = _source_scl[i];
+        for (int i = 0; i < _num_beams; i++) {
```

Comment:
You shouldn't need this block at all.  The call to `configUpdater::instance().subscribe()` will call the `pulsar_grab_callback()` function on registration, which will set these values.